src/openzaak/components/zaken/tests/models/test_zio_block_change.py

```diff
@@ -33,5 +33,5 @@ def test_bulk_update(self):
         )
 
     def test_bulk_create(self):
-        zio = ZaakInformatieObjectFactory.build()
+        zio = ZaakInformatieObjectFactory.build(with_etag=False)
```

Comment:
Similar Question here.

Reply:
Short explanation: it's because of the change in `create_reverse_many_to_one_manager` just like in the comment above. 

Detailed explanation: 
* factory step builder loops through all post_declarations (`with_etag` is also a post_declaration). 
* It tries to get `with_etag` value, fails and tries to raise `AttributeError`
* `AttributeError` message needs `EnkelvoudigInformatieObjectCanonical.__str__`, which uses `latest_version` property
* `latest_version` poperty uses reverse relations and calls `create_reverse_many_to_one_manager` which failes because the instance doesn't have a pk.

In Django 3.2 getting `with_etag` raises `AttributeError` successfully, which is caught later in the builder and the default value is used.